
#offline_mapping Setting
map_in_path: "/home/today/mapin/"
map_out_path: "/home/today/mapout/"
up2down_num: 2    #up_grid_size/down_grid_size，该数值必须为整数倍
lidar_range: 50
frame_sum: 184
up_grid_size: 40


